src/stores/widgets/StopGapWidgetDriver.ts

```diff
@@ -328,6 +333,93 @@ export class StopGapWidgetDriver extends WidgetDriver {
         return { roomId, eventId: r.event_id };
     }
 
+    /**
+     * @experimental Part of MSC4140 & MSC4157
+     */
```

Comment:
Could this have some doc on the params please? For instance, I'm at a loss as to what `parentDelayId ` might be.

Also, what would it mean to pass `null` to `delay` here?

Reply:
Also, this is quite a lot of params for a function. May want to consider making it an object.

Reply:
This implements `WidgetDriver#sendDelayedEvent`, which is present as of v1.8.0 & is where the documentation is.  I'll add a `@see` here to link to it.

Also, making this take an object would require updating the widget-api.  It doesn't take an object for the sake of avoiding adding a new type in case the whole "parent" idea gets dropped.